modules/api_docs/templates/form_api_docs.tpl

```diff
@@ -0,0 +1 @@
+../index.html
```

Comment:
I would highly suggest to move the whole library to `htdocs/js/vendor/swagger-ui/` and just copy the code from `modules/api_docs/static/index.html` inside this tpl to avoid using iframe a lot of unclear references etc.